config/locales/es.yml

```diff
@@ -21,3 +21,39 @@ es:
       welcome_to_daria: "¡Bienvenido a DARIA (edición de %{fund})!"
       what_line: "¿En cuál línea estás trabajando?"
       start: "Empezar"
+  patient:
+    helper:
+      day: "día"
```

Comment:
You said you had some trouble with the pluralizing here, or am I remembering wrong?

Reply:
I was coming in from the wrong direction, coming from the pluralization we had been using. It turns out you can access pluralizing through I18n, though, of course!